lib/Marlin/Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -448,36 +567,28 @@ void GcodeSuite::G28_no_parser(bool always_home_all, bool O, float R, bool S, bo
    */
   #if ENABLED(DUAL_X_CARRIAGE)
 
-    if (dxc_is_duplicating()) {
+    if (idex_is_duplicating()) {
+
+      TERN_(IMPROVE_HOMING_RELIABILITY, saved_motion_state = begin_slow_homing());
 
       // Always home the 2nd (right) extruder first
       active_extruder = 1;
-      homeaxis(X_AXIS
-        #if ENABLED(PRECISE_HOMING)
-          , 0.0f, false, !parser.seen('D')
-        #endif
-      );
+      homeaxis(X_AXIS);
```

Comment:
Looks like PRECISE_HOMING 'D' - do not calibrate feature is dropped here.

Reply:
Intentional. We have no idea or ever tested this on any non-cartesian setup. At the same time, this just creates potential for conflicts with future merges. For these reasons I rather ignore these changes or flatly fail to compile if one attempts to enable it.

I would love to change the prototype of homeaxis to something that doesn't require these changes as an exception, but I will do so in a separate PR.